resources/js/features/events/components/EventAchievementSets/EventAchievementSets.test.tsx

```diff
@@ -5,6 +5,9 @@ import { createAchievement, createEventAchievement, createRaEvent } from '@/test
 
 import { EventAchievementSets } from './EventAchievementSets';
 
+// Suppress "Invalid prop `max` of value `0` supplied to `Progress`".
```

Comment:
Where is this error coming from? We should fix that instead of suppressing it. It's completely valid for a game/event to have no players and show "0 of 0". 

This error also seems to be getting spit out to the console. (see event 11).

Reply:
I confirmed moments ago this is coming from elsewhere - I believe the Achievement Distribution chart. I've removed this comment from this branch and will resolve that issue in a separate PR.